corehq/apps/userreports/models.py

```diff
@@ -143,7 +143,7 @@ class CitusConfig(DocumentSchema):
 
 
 class SQLSettings(DocumentSchema):
-    partition_config = SchemaListProperty(SQLPartition)
+    partition_config = SchemaListProperty(SQLPartition)  # no longer used
```

Comment:
I'm not sure on our policy for deprecating no longer used couch docs. I think there is an issue with removing them here as it would not automatically apply to the database. Meaning docs would have this key specified and (possibly?) be ignored by the ORM?